Makefile

```diff
@@ -13,6 +13,7 @@ MAKE = make
 HAVESYSTEMD = yes
 HAVEOPENRC = no
 HAVEMAN = yes
+HAVEDOC = no
```

Comment:
This looks good to me, but I think the docs should be installed by default. If you go ahead and make that change I will merge this.

Reply:
I have set it to `no` by default as the docs contain the same exact content as the manpages, and normally systemd only ships their manpages in the `man` format. In fact, they are meant to be shipped as manpages only, as they reside in the [`man/`](https://github.com/systemd/systemd/tree/main/man) directory of upstream's source tree, while the actual HTML docs reside in the [`docs/`](https://github.com/systemd/systemd/tree/main/docs) folder.

I think that it'd be better for opensysusers to ship the manpages in `man` format only, and ship the relevant HTML docs, i.e. [`UIDS-GIDS`](https://github.com/systemd/systemd/blob/main/docs/UIDS-GIDS.md) and [`USER_NAMES`](https://github.com/systemd/systemd/blob/main/docs/USER_NAMES.md), as they are referenced in the manpages.

That being said, I'll change this to `yes` by default for the time being, as you requested.

Reply:
You are correct, systemd upstream is not including the html docs by default during install. So I will go ahead with your suggestion of having it disabled. Rever the last commit that enabled it, and I will go ahead and merge.

Reply:
Done